app/controllers/menu_items_controller.rb

```diff
@@ -0,0 +1,37 @@
+class MenuItemsController < ApplicationController
+
+  def add
+    @page_content     = "page-content"
```

Comment:
I would not set the `@page_content`, `@header`, and `@footer` in every controller action, but only where they need to be different.